kong.conf.default

```diff
@@ -1,47 +1,156 @@
+
+# Cluster settings for Kong nodes. Every Kong node that points to the same 
+# database MUST join together to form a Kong Cluster, in both single or multi-DC
+# setups. Kong works on the IP layer (hostnames are not supported, only IPs are 
+# allowed) and it expects a flat network topology without any NAT between the 
+# datacenters. A common setup is having a VPN between the two datacenters such 
+# that the "flat" network assumption of Kong is not violated.
+
+# By default, the cluster_listen address is advertised. If the cluster_listen 
+# host is "0.0.0.0", then the first local, non-loopback, IPv4 address will be 
+# advertised to the other nodes. However, in some cases (specifically NAT 
+# traversal), there may be a routable address that cannot be bound to. This flag
+# enables gossiping a different address to support this.
+# cluster_advertise = NONE
+
+# Key for encrypting network traffic within Kong. Must be a base64-encoded 
+# 16-byte key.
+# cluster_encrypt = NONE
+
+# The TTL (time to live), in seconds, of a node in the cluster when it stops 
+# sending healthcheck pings, possibly caused by a node or network failure. If 
+# the node is not able to send a new healthcheck ping before the expiration, 
+# then new nodes in the cluster will stop attempting to connect to it on 
+# startup. Should be at least 60.
+# cluster_ttl_on_failure = 3600
+
+#################################### DNS #######################################
+
+# By default Kong leverages on dnsmasq to resolve DNS addresses to the upstream
+# services by using the system settings in /etc/hosts and /etc/resolv.conf.
+# dnsmasq = on
+
+# The port used by dnsmasq, only used locally by Kong.
+# dnsmasq_port = 8053
+
+# You can specify and alternate DNS server that Kong will use when proxying
+# requests to the final upstream services. You can't simultaneously specify this
+# setting and have "dnsmasq = on".
+# dns_resolver = NONE
+
+################################## GENERAL #####################################
+
+# The log level for the events returned by Kong and its services.
+# log_level = error
+
+# Comma separated list of additional plugins names to load on this node, used to 
+# load custom plugins that are not already bundled with Kong.
+# Plugins will be loaded from the kong.plugins.{name}.* namespace.
+# custom_plugins = NONE
+
+# The path to the SSL certificate and key that Kong will use when listening on 
+# the proxy_listen_ssl port.
+# ssl_cert = NONE
+# ssl_cert_key = NONE
+
+# Partecipate in the anonymous report program, which sends anonymous data like
+# error stack traces to Mashape, to help improving Kong.
+# anonymous_reports = on
+
+# Nginx configuration parameters that will be dynamically set by Kong.
+# nginx_daemon = on
+# nginx_worker_processes = auto
 
-#anonymous_reports = on
-#mem_cache_size = 128m
+# Allows Kong to set specific connection and proxying settings in Nginx.
+# nginx_optimizations = on
```

Comment:
This actually has no effect right now, the properties it affect - which are commented out right now - need to be benchmarked/debated